tensorrt_llm/_torch/auto_deploy/models/modeling_nemotron_h.py

```
@@ -0,0 +1,543 @@
+        hidden_states = inputs_embeds
+
+        for mixer_block in self.layers:
+            hidden_states = mixer_block(hidden_states)
+
+        hidden_states = self.norm_f(hidden_states)
+
+        return NemotronHOutput(last_hidden_state=hidden_states)
+
+
+class NemotronHForCausalLM(NemotronHPreTrainedModel, GenerationMixin):
+    _tied_weights_keys = ["lm_head.weight"]
```

Ruff (0.14.5)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)